scooby/report.py

```diff
@@ -220,7 +220,7 @@ def __repr__(self):
 
         # Loop over packages
         for name, version in self._packages.items():
-            text += '{:>18} : {}\n'.format(version, name)
+            text += '{:>18} : {}\n'.format(name, version)
```

Comment:
I'm wondering if this should be `{:>21}` to be consistent with the above section. Thoughts, @prisae?

Reply:
Actually, the output visibly lines up when testing locally...

Reply:
Yes, we should be consistent. But this here remains the same, the above should change.